Replicator/Pusher.hh

```diff
@@ -45,13 +46,36 @@ namespace litecore { namespace repl {
         // Checks if a given document should be filtered by this pusher.  The revision body
         // must be loaded prior to this call.
         bool documentShouldBeFiltered(C4Document* doc) const {
+            if(!isDocumentIDAllowed(doc->docID)) {
```

Comment:
Pusher.hh didn't previously have any nontrivial inline methods, so please move these bodies into the .cc file for consistency.